### DISCUSSION TITLE
A Laravel package to onboard your users | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#483] [General]
# A Laravel package to onboard your users | Rias.be

Onboarding your users is one of the most important things you can do in your SaaS or application. After all, first impressions matter!

https://rias.be/blog/laravel-package-to-onboard-your-users

**Comment by tiagofrancafernandes** [comment]
Great!

**Comment by Arden28** [comment]
Great!!

**Comment by emmanuel-ochii** [comment]
@arden28 i would like to know if you have tried using this and how did you go about it..

**Comment by shahsani** [comment]
Wonderful package, indeed I will use it in the current project.
